src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -342,7 +342,7 @@ public OptionData setChannelTypes(@Nonnull ChannelType... channelTypes)
      * @throws IllegalArgumentException
      *         If any of the following checks fail
      *         <ul>
-     *             <li>{@link OptionType type of this option} is {@link OptionType#CHANNEL CHANNEL}</li>
+     *             <li>{@link OptionType type of this option} is {@link OptionType#CHANNEL CHANNEL} and the collection is not empty</li>
```

Comment:
```suggestion
     *             <li>{@link OptionType type of this option} is {@link OptionType#CHANNEL CHANNEL} and the collection is not empty</li>
```
Docs and code are different.

Docs say "collection is empty? IllegalArgumentException"
While the code ignores empty collections.